modules/apps/commerce/commerce-shipment-web/src/main/java/com/liferay/commerce/shipment/web/internal/portlet/action/EditCommerceShipmentMVCActionCommand.java

```diff
@@ -42,6 +42,8 @@
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.component.annotations.Reference;
 
+import static com.liferay.commerce.constants.CommerceShipmentConstants.SHIPMENT_STATUS_PROCESSING;
```

Comment:
Similar here, we should import as `import com.liferay.commerce.constants.CommerceShipmentConstants;`. Then we can use the constant by calling `CommerceShipmentConstants.SHIPMENT_STATUS_PROCESSING;`.